packages/db/src/translate/async_operations.ts

```diff
@@ -20,3 +26,25 @@ export const translateApiAsyncOperationToPostgresAsyncOperation = (
   }
   return translatedRecord;
 };
+
+/**
+ * Generate an API Async Operation record from a PostgreSQL record.
+ *
+ * @param {Object} pgAsyncOperation - A Postgres PDR record
+ * @returns {Object} An Async Operation API record
+ */
+export const translatePostgresAsyncOperationToApiAsyncOperation = (
+  pgAsyncOperation: PostgresAsyncOperation
+): ApiAsyncOperation => {
+  const apiAsyncOperation = {
+    id: pgAsyncOperation.id,
+    description: pgAsyncOperation.description,
+    operationType: pgAsyncOperation.operation_type,
+    status: pgAsyncOperation.status,
+    output: pgAsyncOperation.output ? JSON.stringify(pgAsyncOperation.output) : undefined,
+    taskArn: pgAsyncOperation.task_arn ? pgAsyncOperation.task_arn : undefined,
```

Comment:
```suggestion
    taskArn: pgAsyncOperation.task_arn,
```

I'm pretty sure the conditional here is unnecessary